.github/actions/compose_file/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: compose_file
```

Comment:
I guess the goal of the action is to publish composed xem and example models.

Reply:
you're right, changed in new commit